tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/expressions/binaryoperations/TypeTestExprTest.java

```diff
@@ -736,4 +736,24 @@ public void testTypeTestExpression(String functionName) {
                 "testIntSubtypes"
         };
     }
+
+    @Test
```

Comment:
These tests should also go in `TypeGuardTest`?

Reply:
moved